core/templates/components/state-editor/state-interaction-editor/state-interaction-editor.directive.ts

```diff
@@ -120,7 +120,8 @@ angular.module('oppia').directive('stateInteractionEditor', [
               interactionCustomizationArgs, false);
           };
 
-          var _updateInteractionPreviewAndAnswerChoices = function() {
+          var _updateInteractionPreviewAndAnswerChoices = function(
+              oldToNewListMapping = {}) {
```

Comment:
The naming convention here seems a bit confusing. What are these old and new lists?

Reply:
done.

Reply:
Actually, this is generic functionality, so, as @donosco98 says, I would recommend using a general name (so, there should be no reference to answer groups etc.).

Maybe you can work with @kevintab95 to find a clear name.

Reply:
Oh I see. In that case, we could use the old name. Thanks for clarifying!